vscode-dotnet-runtime-library/src/Utils/WebRequestWorker.ts

```diff
@@ -35,25 +37,47 @@ export class WebRequestWorker {
         return this.cachedData;
     }
 
+    /**
+     *
+     * @param url The url containing raw json data to parse.
+     * @returns a serizled JSON object.
+     */
+    public async fetchJsonObjectFromUrl(url : string)
```

Comment:
nit: might be more clear to use types here, so that the method signature might look something like:
> public async fetchJsonObjectFromUrl<T>(url : string): T

and then you could try to cast the JSON object to that type before returning. Not a big deal, just might make reading a little clearer

Also in the comment: serizled => serialized

Reply:
This method has been removed, but I think it's a really good idea to use actual JSON objects in this case too. Unfortunately the structure of the JSON could change pretty frequently, so a lot of this becomes <any>